src/luaconnector/common.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <thread>
+#include <mutex>
+#include <vector>
+#include <string>
+#include <cstdio>
+#include <utility>
+#include <deque>
+#include <chrono>
+
+#ifdef _WIN32
+#ifndef _WIN32_WINNT
+#define _WIN32_WINNT 0x0A00
+#endif
+#endif
+
+#ifndef ASIO_STANDALONE
+#define ASIO_STANDALONE
+#endif
```

Comment:
This is supposed to be handled by the Makefile. This breaks using your code with boost::asio